docs/docs/sourcing-from-ghost.md

```diff
@@ -62,43 +62,63 @@ Once the source plugin is set up, you can use the `createPages` API in `gatsby-n
 There are several ways to structure queries depending on how you prefer to work, but here's a very minimal example:
 
 ```javascript:title=gatsby-node.js
+const _ = require(`lodash`)
+const Promise = require(`bluebird`)
```

Comment:
Have also removed this requirement https://github.com/TryGhost/gatsby-starter-ghost/commit/a24d7bdcd875eb44ed5d20f65a65029c262ce9bc

Reply:
Thank you so much @ErisDS .